src/wcs2_0/src/test/java/org/geoserver/wcs2_0/kvp/GetCapabilitiesTest.java

```diff
@@ -125,6 +125,35 @@ public void testAcceptVersions() throws Exception {
                 "0", "count(//ows:ExceptionReport//ows:Exception[@locator='wCS'])", dom);
     }
 
+    @Test
+    public void testInternationalContent() throws Exception {
+        GeoServerInfo global = getGeoServer().getGlobal();
+        global.getSettings().setProxyBaseUrl("src/test/resources/geoserver");
+        getGeoServer().save(global);
+        Catalog catalog = getCatalog();
+        CoverageInfo ci = catalog.getCoverageByName(getLayerId(TASMANIA_DEM));
+        ci.setTitle("My Title");
+        ci.setAbstract("My Abstract");
+        ci.getKeywords().add(0, new Keyword("my_keyword"));
+        MetadataLinkInfo mdl1 = catalog.getFactory().createMetadataLink();
+        mdl1.setContent("http://www.geoserver.org/tasmania/dem.xml");
+        mdl1.setAbout("http://www.geoserver.org");
+        ci.getMetadataLinks().add(mdl1);
+        MetadataLinkInfo mdl2 = catalog.getFactory().createMetadataLink();
+        mdl2.setContent("/metadata?key=value");
+        mdl2.setAbout("http://www.geoserver.org");
+        ci.getMetadataLinks().add(mdl2);
+        catalog.save(ci);
+        Document dom =
+                getAsDOM("wcs?service=WCS&version=2.0.1&request=GetCapabilities&Language=ita");
+
+        checkValidationErrors(dom, getWcs20Schema());
+        String base = "//wcs:Capabilities/wcs:Contents/wcs:CoverageSummary[wcs:CoverageId = '";
+        base += getLayerId(TASMANIA_DEM).replace(":", "__") + "']/";
+        assertXpathEvaluatesTo("My Title", base + "ows:Title", dom);
+        assertXpathEvaluatesTo("My Abstract", base + "ows:Abstract", dom);
```

Comment:
there are no assertions for the urls in the getCapabilities response that should show the Language parameter.